public/projects/pair_programming.md

```diff
@@ -0,0 +1,103 @@
+# Anonymous Pair Programming
+
+For this project you will find another person to pair with within programmer communities.
+
+There are several communities that you can use to find pairs within.
+
+- [Agile Ventures Pair Group](https://www.agileventures.org/pair "Agile Ventures Pairing Site Link")
+- [Reactiflux Discord Group](https://www.reactiflux.com/ "Reactiflux Discord Group")
+- [Code Together Reddit](https://www.reddit.com/r/codetogether/ "Code Together Reddit")
+
+## Steps
+
+### Install Tech for Video/Voice/Screen Sharing
+
+* You will need at a minimum one of the video & screen share options such as Zoom, Hangouts, or Skype.
+* **Opionally** you can install an editor extension to allow for collaboration within the editor for two people at once.
+
+### Find a Pair
+
+Use one of the communities listed above in the description to find a pair or mentor who is willing to work on the project you are developing.
+
+* Agile Ventures
+  1. Create an account at [Agile Ventures Sign Up](https://www.agileventures.org/users/sign_up)
+  2. Sign up for the Agile Ventures Slack group
+  3. Review the projects and join if you are interested
+  4. Ask if anyone is interested in pairing with you on a personal or Agile Ventures project
+
+* Reactiflux
+  1. Sign up for Discord and optionally install their desktop application at [Discord](https://discordapp.com/ "Discord Sign Up")
+  2. Sign up for the Reactiflux Discord Group at [Reactiflux Sign Up](https://www.reactiflux.com/ "Reactiflux Discord Sign Up")
+  3. Introduce yourself within the `#introductions` chanel
+  4. Ask if anyone is willing to pair on a project within the `#need-help` channel
+
+* Code Together Reddit
+  1. Sign up for a Reddit account at [Sign up](https://www.reddit.com/register "Reddit Sign Up")
+  2. Subscribe to the [CodeTogether Sub-reddit](https://www.reddit.com/r/codetogether/ "CodeTogether Sub-reddit")
+  3. Introduce yourself and your project in a new post and describe the help you are looking for
+  4. Be open to accepting help and using the tools that your pair may be familiar with
+
+### Schedule a Time
+
+1. Introduce yourself to the community
+2. Explain that you are a beginner and are looking for some help working on one of your projects
+3. Ask whether anyone would like to help you for an hour or so.
+4. Either invite the person who agrees to help you using a calender invitation, or just start if they are ready.
+
+### Work on a Suggested Project
+
+1. Pick one of the projects from the BTVCA curriculum if you can.
+2. If you have finished your assignments or are in between projects see if your pair has a project to work on.
+3. Have a plan of what you want to achieve in the pair programming session. Have an end in mind.
```

Comment:
this is a nitpick, but #2 isn't really a separate step from #1; maybe these should be bullets instead of numbers